test/e2e/app-dir/metadata-dynamic-routes/index.test.ts

```diff
@@ -444,12 +444,8 @@ describe('app dir - metadata dynamic routes', () => {
       // absolute urls
       ogImageUrlPattern = /https:\/\/[\w-]+.vercel.app\/opengraph-image\?/
       twitterImageUrlPattern = /https:\/\/[\w-]+.vercel.app\/twitter-image\?/
-    } else if (isNextStart) {
-      // configured metadataBase for next start
-      ogImageUrlPattern = /https:\/\/mydomain.com\/opengraph-image\?/
-      twitterImageUrlPattern = /https:\/\/mydomain.com\/twitter-image\?/
     } else {
-      // localhost for dev
+      // localhost if not explicitly configured. Should this error?
```

Comment:
There's a warning but only in production. We discussed many times back and forth around it, want to make it easy since the direct deployment can work on vercel without configuration, no so warning in dev
```suggestion
      // localhost if not explicitly configured
```